tests/docker_images/linux-loongarch64/ubuntu-x-tools/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+FROM ubuntu:latest
```

Comment:
would it be beneficial to pin to an LTS version?

Reply:
looks like the other dockerfiles have been updated to derive `FROM ubuntu:22.04`

Reply:
I somehow missed this one. Will fix.